catalog-info.yaml

```diff
@@ -648,13 +648,57 @@ spec:
           build.pull_request.id == null || (build.creator.name == 'elasticmachine' && build.pull_request.id != null)
       repository: elastic/beats
       cancel_intermediate_builds: true
-      cancel_intermediate_builds_branch_filter: "!main !7.17 !8.*"
+      cancel_intermediate_builds_branch_filter: "!7.17"
       skip_intermediate_builds: true
-      skip_intermediate_builds_branch_filter: "!main !7.17 !8.*"
+      skip_intermediate_builds_branch_filter: "!7.17"
       # env:
       #   ELASTIC_PR_COMMENTS_ENABLED: "true"            #TODO: uncomment after tests
       teams:
         ingest-fp:
           access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-elastic-agent-package-7-17
+  description: Buildkite pipeline for packaging Elastic Agent v7.17 package
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/beats-xpack-elastic-agent-package-7-17
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: elastic-agent-package-7-17
+      description: Buildkite pipeline for packaging Elastic Agent package
+    spec:
+      repository: elastic/elastic-agent
+      pipeline_file: ".buildkite/x-pack/elastic-agent/pipeline.xpack.elastic-agent.package.yml"
+      provider_settings:
```

Comment:
> @v1v the pipeline will be triggered remotely. Removed other branches from configuration just in case

If that's the case, what's the reason for using all those settings in the `provider_settings:` rather than:

```
      provider_settings:
        trigger_mode: none # don't trigger jobs from github activity
```

See https://github.com/elastic/beats/blob/051d06c17ac7a311649c022da31a5884d3e934f1/catalog-info.yaml#L90C1-L91C69

and https://registry.terraform.io/providers/buildkite/buildkite/latest/docs/resources/pipeline#trigger_mode

Reply:
Good question. I've seen this example before and it is confusing. It uses the `trigger_mode: none` and all other settings as well. I'd use just `trigger_mode: none` without all other settings for github

Reply:
Then probably we don't need `cancel_intermediate_builds` and `skip_intermediate_builds`

Reply:
IIRC, you can configure the branch where the build can run to, see https://buildkite.com/docs/pipelines/trigger-step#environment-variables